src/sentry/conf/server.py

```diff
@@ -133,6 +133,8 @@ def env(key, default='', type=None):
     if url.scheme == 'mysql':
         DATABASES['default']['ENGINE'] = 'django.db.backends.mysql'
 
+ALLOWED_HOSTS = ['localhost']
```

Comment:
Excellent catch. According to the docs localhost should be added if we're running in DEBUG mode. I noticed that when running `lims devserver` the configuration is run before running the tool, so this DEBUG flag wasn't set correctly. So I fixed that.

However, then I also noticed that even if we do set it correctly, localhost isn't being added for some reason as it should be (also in 1.9). Rather than digging too deep into that at the moment, I'm just adding it when we are in DEBUG mode: https://github.com/commonlims/commonlims/pull/44/commits/4dbcbc7ba2efcd7404a7c5538462eaea2c77a394